host-configs/juno-armv8-gcc@5.2.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
As part of your PR, can you increment the license year to 2021 in all your files?

Reply:
Done.